src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -954,6 +954,42 @@ private void SetDetailFieldsForCreation(PurchaseOrderDetail detail, int newOrder
 
         private void UpdateDeliveries(PurchaseOrderDetail purchaseOrderDetail, decimal exchangeRate, int supplierId)
         {
+            if (purchaseOrderDetail.OrderQty > purchaseOrderDetail.OurQty)
```

Comment:
I don't understand this? I thought from what Andrew said this is to do with keeping deliveries in sync with the order line. I don't think it's anything to do with order qty and our qty. 

Also it presumably can't just be about adding a delivery. What about the scenario where they reduce the qty (our qty) on the line?

Reply:
I haven't seen the ticket so this is maybe a different issue from the one that Andrew mentioned this morning?

Reply:
It doesn't seem to address the 0 value problem either

Reply:
Andrew said that they never reduce the qty, it's just not a thing that they do. Asked about the scenario but he was pretty certain it shouldn't ever happen.

Reply:
maybe they shouldn't but it looks like that's what's happened in the example that Andrew posted. 
If it's not supposed to happen then we should disallow it

Reply:
Yeah I can change this on the front end. @lewisrenfrew Any thoughts on this? I know we discussed a few different approaches.

Reply:
I would just throw an exception

Reply:
Yeah, ok definitely. General approach lines up with what you expected though with the use of the Qty fields?